config/environment.js

```diff
@@ -47,5 +47,6 @@ module.exports = function (environment) {
     // here you can enable a production-specific feature
   }
 
+  ENV.BASE_API_URL = 'https://staging-api.realdevsquad.com';
```

Comment:
This will override any base URL in any environment.
Better keep it in a separate `if` block for the `staging` environment.

Reply:
But why should we be worried about about frontend being on staging/production?
The API base URL would be same even when our frontend is on staging or production. What do you say?

Reply:
When we host the backend on production, we'll use the production URL on our `Production Frontend`
Same for staging and testing

Reply:
Even when the frontend is on staging or production, the backend API endpoint would remain same for all, right? Or am I missing something? 🤔

Reply:
No, we'll have different backend servers and Databases